6-Final-Proj/client/src/components/TodoItem/TodoItem.tsx

```diff
@@ -70,7 +68,7 @@ export const TodoItem: FC<ITodoItem> = ({
   const editButton = (
     <Button
       icon="&#xe3c9;"
-      additionalClasses={[classes.editBtn]}
+      additionalClasses={classes.editBtn}
```

Comment:
since it's not a native html element, you might just call this prop `className` and add to the classes list